security-indicators/base/security-indicators.yaml

```diff
@@ -0,0 +1,193 @@
+---
```

Comment:
this should be in openshift folder

Reply:
do we do that in other apps?

Reply:
one step back then, why is it an app? I thought this is a workflow triggered by another component with logic to do that. Currently, it is not an app right?

for workflow component we use openshift and argo workflows folders in general.

Reply:
ja, right, which would be the other component that wants to create an SI workflow?

Reply:
<component>-> learn-packages workflow -> solver workflow + SI workflow once graph-sync-solver is completed and checking if that package is already stored on the database.

with empty thoth:
- <component> == init-job initially after provision of thoth uses learn-packages workflow
- <component> == graph-refresh-job uses learn-packages workflow

with thoth with data:
- <component> == graph-refresh-job uses SI workflow for packages solved with no SI evaluate

New tasks:
- [ ] graph schema to sync SI.
- [ ] sync logic in storages for SI.
- [ ] add graph-sync-security-indicator task to SI.
- [ ] create methods to query for SI for a certain package version index.
- [ ] New learn-packages workflow make by solver workflow + si workflow scheduled if some conditions are met.
- [ ] New logic in graph-refresh-job for learn-packages workflow for new knowledge.
- [ ] New logic in graph-refresh-job for si workflow for knowledge already collected.

Reply:
I think there does not need to be any logic in init-job. The graph refresh should handle the case when the package is solved but has no si. Otherwise the same logic will be at two different places, but the first one is basically a subset of the latter one.

Reply:
But initi-job logic won't be completely changed, just the workflow to be scheduled, instead of solvers, we have solvers + SI after provision and we can run integrations tests with advises also from the security point of view?

Reply:
If you mean the one workflow per each package scheduled, that sounds good.

Reply:
Yes exactly :)